webapp/src/main/webapp/templates/freemarker/body/partials/individual/propStatement-dataDefault.ftl

```diff
@@ -15,7 +15,13 @@
 <@showStatement statement property datatype />
 
 <#macro showStatement statement property datatype>
+
     <#assign theValue = statement.value />
+
+<#-- create a weblink if the property range is a URI AND it contains http -->
+    <#if property.rangeDatatypeURI?? && theValue?contains("http")>
+        <#assign theValue = "<a href=\"" + statement.value + "\" target=\"_blank\">" + statement.value + "</a>" />
```

Comment:
Would be good to add an "external" class to the link so that it can be decorated in CSS to indicate it is (probably) an external link.

Reply:
Currently, no. But then there are also few places where external links are generated, and it's relatively easy for CSS designers to target those instances specifically to decorate them if desired.

By linking URIs based on the recognised datatype, they could potentially occur anywhere in the ontologies. As such, the only way someone could decorate them - and not the application URIs - is if there is a CSS class.

Reply:
@grahamtriggs I was looking into your point about making sure the changes are also in the Vitro file. I was wondering why there is a VIVO version. In examination it looks like there are only a few differences, all of which you made about 2 years ago in https://jira.duraspace.org/browse/VIVO-1237.  Can you explain why these changes can just be made in the Vitro file such that we can abandon the VIVO file?

Reply:
The changes for the citation tags was not made in the Vitro version, because outputting the tags requires knowledge of the ontologies that you are working with (which Vitro can't assume).

However, on reflection it is probably the case that a lib-meta-tags.ftl with a no-op macro could be placed in Vitro, being overridden by the ontology aware version that is in Vivo, and the propStatement-dataDefault.ftl changes moved into Vitro.

Reply:
So it sounds like I should copy https://github.com/vivo-project/VIVO/blob/develop/webapp/src/main/webapp/templates/freemarker/lib/lib-meta-tags.ftl to the Vitro location and then do .... what -- change the switch statement? Can we just leave the cases in the switch as is? Not sure what the harm is for leaving it as is in Vitro, if the ontology isn't there the switches will never be fullfilled. Right?
Then, when we resolve above I can create a Vitro fork and test that in our 1.9.3 CU Boulder dev environment.
After that would I just copy the propStatement-dataDefault.ftl from Vivo to Vitro and then remove the file from VIVO?

Reply:
can anybody confirm my steps above? THis would mean that lib-meta-tags.ftl in Vitro does contain elements from the VIVO ontologies, but I'm not sure this is a big deal except from a purist standpoint. It would eliminate code bloat and redundancy to do it this way. IMHO it's best shrink the code best instead of expanding it to reduce complexity.